lib/transactions/maltego/nettacker_transforms/src/nettacker_transforms/transforms/http_cookie_vuln_scan.py

```diff
@@ -0,0 +1,49 @@
+import random
+
+from canari.maltego.transform import Transform
+from canari.maltego.entities import URL
+from canari.framework import EnableDebugWindow
+from common.entities import NettackerScan
+
+from lib.vuln.cookie.engine import start
+
+from database.db import __logs_by_scan_id as find_log
+
+__author__ = 'Shaddy Garg'
+__copyright__ = 'Copyright 2018, nettacker_transforms Project'
+__credits__ = []
+
+__license__ = 'GPLv3'
+__version__ = '0.1'
+__maintainer__ = 'Shaddy Garg'
+__email__ = 'shaddygarg1@gmail.com'
+__status__ = 'Development'
+
+
+@EnableDebugWindow
+class HTTP_CookieVulnScan(Transform):
+    """TODO: Your transform description."""
+
+    # The transform input entity type.
+    input_type = NettackerScan
+
+    def do_transform(self, request, response, config):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5181770)